humans/tasks.py

```diff
@@ -71,7 +78,9 @@ def update_public_keys():
                        "humans/emails/fingerprint_changed.txt")
             user.keyserver_url = ""
             user.save()
-        elif state == "valid":
+        elif state[0] == "valid":
+            # Checks if key is about to expire
+            check_key_expiration_date(state[1])
```

Comment:
As said in this comment: https://github.com/whitesmith/hawkpost/pull/274#discussion_r172216886

This should be a separate task, since this task is design to update profile that added a `keyserver_url` and doesn't check all the keys in the system.